src/Agent.Worker/ExecutionContext.cs

```diff
@@ -102,8 +103,8 @@ public sealed class ExecutionContext : AgentService, IExecutionContext
         public HashSet<string> OutputVariables => _outputvariables;
         public bool WriteDebug { get; private set; }
         public List<string> PrependPath { get; private set; }
-        public ContainerInfo Container { get; private set; }
-        public List<ContainerInfo> SidecarContainers { get; private set; }
+        public ContainerInfo Container { get; set; }
+        public List<ContainerInfo> SidecarContainers { get; set; }
```

Comment:
why do the setters need to be public?

Reply:
Yeah, had changed those to test something, changing it back.